samples/brokers/opcua-monitoring-broker/opcua-monitoring-broker.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Grpc.AspNetCore" Version="2.40.0" />
-    <PackageReference Include="OPCFoundation.NetStandard.Opc.Ua" Version="1.4.367.42" />
+    <PackageReference Include="OPCFoundation.NetStandard.Opc.Ua" Version="1.4.368.58" />

```

Comment:
client should only need OPCFoundation.NetStandard.Opc.Ua.Client to reduce overall dependencies.

Reply:
latest is 1.4.370.1